DOTNET-example/README.md

```diff
@@ -14,23 +14,11 @@ will eventually export the trace data to our UI.
 * A VMware Aria Operations for Applications account (formerly known as Tanzu Observability by Wavefront) account, which gives you access to a cluster. If you don’t have a cluster, [sign up for a free trial](https://www.vmware.com/products/aria-operations-for-applications.html).
 * Clone the [OpenTelemetry Examples](https://github.com/wavefrontHQ/opentelemetry-examples) repository.
 * Install the Docker platform. You’ll run the Wavefront proxy on Docker for this tutorial.
-* Install the Wavefront proxy on Docker    
-    ```
-    docker run -d \
-        -e WAVEFRONT_URL=https://{INSTANCE_NAME}.wavefront.com/api/ \
-        -e WAVEFRONT_TOKEN={TOKEN} \
-        -e JAVA_HEAP_USAGE=512m \
-        -e WAVEFRONT_PROXY_ARGS="--customTracingListenerPorts 30001" \
-        -p 2878:2878 \
-        -p 30001:30001 \
-        wavefronthq/proxy:latest
-    ```
-    Replace:
-    * `{INSTANCE_NAME}` with the product instance (for example, https://longboard.wavefront.com).
-    * `{TOKEN}` with the API token linked to an account with Proxy permission.
-      See [Generating and an API Token](https://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
-    
-    See [Install a Proxy](http://docs.wavefront.com/proxies_installing.html#install-a-proxy) to find other options for installing the proxy on your environment.
+* [Install the Wavefront proxy](http://docs.wavefront.com/proxies_installing.html#install-a-proxy). You need to add the following configurations to the default proxy installation configurations.
+  ```
+  -e WAVEFRONT_PROXY_ARGS="--otlpGrpcListenerPorts 4317" \
+  -p 4317:4317 \  
+  ```
```

Comment:
If we're keeping Docker as a requirement for these examples (which I don't think is necessarily a bad idea since it makes it easy for non-Java developers to get it running), I think we should instead link to https://docs.wavefront.com/proxies_container.html#run-a-proxy-in-a-docker-container.

That link gives examples of how to run the Proxy via Docker. I think our specific config could instead be replaced with:
> When running the Proxy in Docker, be sure that the `WAVEFRONT_PROXY_ARGS` environment variable contains `--otlpGrpcListenerPorts 4317`, and expose the OpenTelemetry port via `-p 4317:4317`.

Reply:
Thank you! 
Yes , the above only works for Docker, right? I will make it generic so users know what to add.

Reply:
I will make these changes in all the examples.